docs/help/macos-login-shells.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: macOS Login Shells
+description: |-
+  In other unix systems terminal emulators are generally started
+  using non-login shells. Due to various reasons, macOS defaults 
+  all shells as login shells.
+---
+
```

Comment:
Not sure if its intended, but the text here seems to have a different color than the rest of the text below
![image](https://github.com/user-attachments/assets/30d9aeaf-1497-42e0-9d42-fba1b6c366bb)

Reply:
I didn't change any color settings or anything, and I'm not sure what text your referencing, it all looks normal to me

Reply:
ahh, unless you're referring to the content, in which case yes. I need to replace shells with terminals in the brief. good catch